FireboltDotNetSdk.Tests/Unit/FireboltCommandTest.cs

```diff
@@ -188,5 +189,64 @@ public void SetParamListDatesTest()
             var expect = commandText.ToString("yyyy-MM-dd HH:mm:ss");
             Assert.That("'" + expect + "'", Is.EqualTo(result));
         }
+
+        [Test]
+        public void TimestampTzTest()
+        {
+            var responseWithTimestampTz =
+                "{\"query\":{\"query_id\": \"1739956EA85D7645\"},\"meta\":[{\"name\": \"CAST('2022-05-10 23:01:02.12345 Europe\\/Berlin' AS timestamptz)\",\"type\": \"TimestampTz\"}],\"data\":[[\"2022-05-10 21:01:02.12345+00\"]],\"rows\": 1,\"statistics\":{\"elapsed\": 0.001312549,\"rows_read\": 1,\"bytes_read\": 1,\"time_before_execution\": 0.000553908,\"time_to_execute\": 0.000173431,\"scanned_bytes_cache\": 0,\"scanned_bytes_storage\": 0}}";
```

Comment:
I would also suggest adding tests for non-unix timestamp values for timestamptz and timestampntz values.

Reply:
Sure added it to https://github.com/firebolt-db/firebolt-net-sdk/blob/bf0c8797bf9ce0cc1b53765365e2cf987bd6cb14/FireboltDotNetSdk.Tests/Unit/FireboltCommandTest.cs#L241-L260